packages/esm-patient-registration-app/src/patient-registration/field/dob/dob.component.tsx

```diff
@@ -58,12 +58,20 @@ export const DobField: React.FC = () => {
   const onEstimatedMonthsChange = (e) => {
     const months = +e.target.value;
 
-    if (!isNaN(months) && months <= 11 && months >= 0) {
+    if (!isNaN(months)) {
       setFieldValue('monthsEstimated', months);
       setFieldValue('birthdate', calcBirthdate(yearsEstimateMeta.value, months, dateOfBirth));
     }
   };
 
+  const calculateBirthdate = () => {
```

Comment:
There seem to be already a `calcBirthdate()` function. Isn't this a bit confusing?

Reply:
I agree